kbase-extension/static/kbase/js/util/jobLogViewer.js

```diff
@@ -571,20 +582,20 @@ define([
          * There's something minor that I'm missing, I think, about how the scrolling gets
          * managed.
          * @param {ScrollEvent} e
-        function handlePanelScrolling(e) {
+        handlePanelScrolling(e) {
```

Comment:
It's been long enough, and there's been enough revisions, that we can probably consign this hunk of comment to `/dev/null`